frontend/src/lib/components/LemonBubble/LemonBubble.scss

```diff
@@ -0,0 +1,93 @@
+@import '~/vars';
+
+$lemon-counter-small: 1.2rem;
+$lemon-counter-font-small: 0.6rem;
+$sf-small: 0.8;
+$sf-large: 1.2;
+$position-offset: 0.5rem;
+
+.LemonBubble {
+    flex-shrink: 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: var(--white);
+    background: var(--primary);
+    border: 2px solid var(--bg-light);
+    width: $lemon-counter-small;
+    height: $lemon-counter-small;
+    font-size: $lemon-counter-font-small;
```

Comment:
Hmm, I'm not sure but I thought this should also have `font-weight: 600`?

Reply:
Figma says 700 so changed it to match that 👍